src/EncryptionBehavior.php

```diff
@@ -332,9 +334,18 @@ protected static function insertEncryptedColumnsDeclaration(string &$script, int
      */
     protected function addEncryptionToSetter(string &$script, string $columnPhpName, bool $isSearchable): void
     {
+        /** @var int|bool $setterLocation */
```

Comment:
This is wrong, it can never be true.
Also, you either use annotations or exceptions, using both is not neeeded